src/main/java/com/greenfox/poker/service/GameService.java

```diff
@@ -1,10 +1,48 @@
 package com.greenfox.poker.service;
 
+import com.greenfox.poker.model.Game;
+import com.greenfox.poker.model.GameState;
+import com.greenfox.poker.model.GameInfoList;
+import com.greenfox.poker.repository.GameRepo;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class GameService {
 
+  @Autowired
+  GameRepo gameRepo;
 
+  @Autowired
+  GameInfoList gameInfoList;
 
-}
+  public Game saveGame(Game game){
+    Game newGame = new Game();
```

Comment:
If it's just saving, then why do you create a new Game?
If it's not just saving, then you should improve the naming.
Otherwise, you should use dependency injection here instead of instantiation.